.github/workflows/main.yml

```diff
@@ -50,29 +50,6 @@ jobs:
           version=$(npm version patch)
           echo "::set-output name=version::$version"
 
-      - name: Push to temp branch
-        id: push-temp
-        if: ${{ steps.patch.outcome == 'success' }}
-        run: git push origin HEAD:temp-branch --follow-tags
-
-      - name: Create Pull Request
-        id: create_pr
-        if: ${{ steps.push-temp.outcome == 'success'}}
-        uses: peter-evans/create-pull-request@v3
-        with:
-          branch: temp-branch
-          delete-branch: true
-          reviewers: 'dpatton1992'
-          committer: 'svc-gh-actions <svc_gh_actions@tilled.com>'
-          author: 'svc-gh-actions <svc_gh_actions@tilled.com>'
-          title: '${{ steps.patch.outputs.version }}'
-          body: 'Automated PR for ${{ steps.patch.outputs.version }}'
-          labels: 'automated'
-
-      - name: Enable Pull Request Automerge
-        if: ${{ steps.create_pr.outputs.pull-request-number }}
-        run: gh pr merge --merge --auto "${{ steps.create_pr.outputs.pull-request-number }}"
```

Comment:
Did we intend to leave this off of the block of code that was moved down?

Reply:
This command actually does not currently work. See [here](https://github.com/gettilled/tilled-node/actions/runs/9564589124/job/26365664569). Essentially, it needs an additional env variable to use the github CLI. I could refactor it to a git command or add the variable, but I don't see a ton of value in enabling automerge on a PR that still requires a review.